pkg/timeutil/timeutil.go

```diff
@@ -26,3 +42,22 @@ func LastWeekday(weekday time.Weekday) time.Time {
 
 	return date
 }
+
+// StringToWeekday maps a string to a time.Weekday.
+func StringToWeekday(day string) (time.Weekday, error) {
+	timeWeekday, ok := weekday[strings.ToLower(day)]
+	if !ok {
+		return time.Sunday, fmt.Errorf("timeutil: Invalid weekday '%s' given", day)
+	}
+
+	return timeWeekday, nil
+}
+
+// FormatDuration prints the time using the given format.
+func FormatDuration(duration time.Duration, timeFormat string) string {
+	if timeFormat == "decimal" {
```

Comment:
I believe that TOML uses the text unmarshaller/marshaller interface. Since the TimeFormat doesn't have that declared, it should happily inherit it from the `string` type. I think this method is only useful if the type is defined in the Config struct as Ioannis said